packages/terra-alert/tests/wdio/alert-spec.js

```diff
@@ -62,12 +62,25 @@ Terra.describeViewports('Alert', ['tiny', 'large'], () => {
   });
 
   describe('Critical alert notification banner', () => {
-    it('alert content is focused when rendered with an action element', () => {
+    it('alert content is focused when rendered with an action element and dismiss', () => {
       browser.url('/raw/tests/cerner-terra-core-docs/alert/custom-prop-alert');
-
       browser.keys(['Tab', 'Tab', 'Tab', 'Tab', 'Enter']);
 
-      Terra.validates.element('alert focused');
+      Terra.validates.element('alert focused with action and dismiss');
+    });
+
+    it('alert content is focused when rendered with only action element', () => {
+      browser.url('/raw/tests/cerner-terra-core-docs/alert/custom-prop-alert');
+      browser.keys(['Tab', 'Tab', 'Tab', 'Down arrow', 'Down arrow', 'Down arrow', 'Down arrow', 'Down arrow', 'Down arrow', 'Enter', 'Escape', 'Tab', 'Enter']);
+
+      Terra.validates.element('alert focused with action');
```

Comment:
Can we explicitly check for focus, e.g:
https://github.com/cerner/terra-framework/blob/18ce46e82c0dc60d1665700a2453bb92ed9df6c7/packages/terra-slide-panel/tests/wdio/slide-panel-spec.js#LL191C14-L191C14

Reply:
Added focus check here https://github.com/cerner/terra-core/pull/3823/commits/d4700b8f71c7da3a79709a02b3651a0419dcdd71#diff-4310e0b70d7468d01c489f156612886bbb0be04b7579c1bd7bb2583723e68521